platform/linux-generic/odp_packet.c

```diff
@@ -1259,6 +1259,22 @@ int odp_packet_l4_offset_set(odp_packet_t pkt, uint32_t offset)
 	return 0;
 }
 
+void odp_packet_l3_chksum_insert(odp_packet_t pkt, int l3)
+{
+	odp_packet_hdr_t *pkt_hdr = packet_hdr(pkt);
+
+	pkt_hdr->p.output_flags.l3_chksum_set = 1;
+	pkt_hdr->p.output_flags.l3_chksum = l3;
+}
+
+void odp_packet_l4_chksum_insert(odp_packet_t pkt, int l4)
```

Comment:
maybe it's reasonable to change api at the same time to:
void odp_packet_l3_chksum_insert(odp_packet_t pkt,  odp_bool_t enable)
?
this int l3, int l4 is confusion.
@psavol ?

Reply:
Agree with @muvarov 

Perhaps rework this in light of PR #264?